src/features/wallet/TwoKeyContractModal.tsx

```diff
@@ -24,8 +38,43 @@ const _TwoKeyContractModal: ForwardRefRenderFunction<
   { isOpen, close, hotAddress, votingAddress, isConnected, network, signer },
   externalRef
 ) => {
+  const [showForm, setShowForm] = useState(false);
+  const [value, setValue] = useState("");
+  const [error, setError] = useState("");
+  const [success, setSuccess] = useState(false);
+  const closeForm = useCallback(() => {
+    if (showForm) {
+      setShowForm(false);
+      setValue("");
+    } else {
+      setSuccess(false);
+      setShowForm(true);
```

Comment:
is `closeForm` meant to toggle the form on and off?

Reply:
Yeah if you look at the design, you open the form through what  looks like a anchor tag

https://old-star.surge.sh/#